### DISCUSSION TITLE
Deduplicate reads from 10x fragment.tsv.gz

### DISCUSSION THREAD

**Discussion by vikon72** [GreenleafLab/ArchR#1989] [Feature Requests]
hi ,there
  I found that there was no dedup in the 10x fragment file. It includes the read pair marked unique and all duplicate read pairs. Whether it leads to calculations of gene scores exists error?

**Comment by rcorces** [comment]
I'm afraid your interpretation of the fragment file is incorrect.
https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/output/fragments
The 4th column is
"The total number of read pairs associated with this fragment. This includes the read pair marked unique and all duplicate read pairs"

**Comment by vikon72** [comment]
hi,
  Thanks for your reply.But In my sight，it just means Total read pais includes all reads pairs,How do you understand that?
In addition，this is a part of my 10X fragment file.It was acquired by cellranger-atac 2.1.0.The same cut sites appears more than once.
It was changed a lot between cellranger-atac 1.1.0 and cellranger-atac 2.1.0.Some functions In ArchR maybe effective  no longer.such as
`- getValidBarcodes()`. Is there an update plan？

> chr1    181470  181519  AATGCCATCGAGAAGC-1      2
> chr1    181470  181519  ACCAAACGTGCCCTAG-1      4
> chr1    181470  181519  CAATCCCCAAGTTGCT-1      2
> chr1    181470  181519  CACCTTGGTATTGTCG-1      2
> chr1    181470  181519  TAGTCCCTCTCCCTTT-1      1
> chr1    181470  181520  AAATGAGAGTACCACT-1      2
> chr1    181470  181520  ACCGCAGTCACGTCAA-1      1
> chr1    181470  181520  CAGTATGAGCATACCT-1      1
> chr1    181470  181521  AAATGCCGTATTCACG-1      1
> chr1    181470  181521  ACATGGTGTATTCACG-1      5
> chr1    181470  181521  ATTTGTCGTCTGGTTA-1      3
> chr1    181470  181521  CCGTACTTCTGCCGAG-1      1
> chr1    181470  181521  CTCTACGGTGATCAGG-1      3
> chr1    181470  181521  TCAATTCAGCCCATGC-1      3
> chr1    181470  181521  TGCATGATCGCGTTCT-1      2

Thanks!
